nav2_smac_planner/lattice_primitives/generate_motion_primitives.py

```diff
@@ -0,0 +1,246 @@
+# Copyright (c) 2021, Matthew Booker 
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License. Reserved.
+
+import argparse
+import json
+import logging
+import time
+from datetime import datetime
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from lattice_generator import LatticeGenerator
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+VERSION = 1.0
```

Comment:
Can this be set in another file in the root of lattice_primitives that is imported and used? That way if/when we update this, its clear where this is and so we can increment it (and be globally available if we need to support backports of old versions)